vyper/semantics/analysis/local.py

```diff
@@ -811,7 +811,10 @@ def visit_Call(self, node: vy_ast.Call, typ: VyperType) -> None:
             for arg, arg_type in zip(node.args, func_type.arg_types):
                 self.visit(arg, arg_type)
         else:
-            # builtin functions
+            # builtin functions and interfaces
+            if self.function_analyzer and hasattr(func_type, "mutability"):
```

Comment:
builtins always have the "mutability" thing right?

Reply:
yes, but not interface functions

Reply:
oh I think they are wrapped in a TYPE_T